src/gridfs/download.ts

```diff
@@ -56,15 +56,41 @@ export interface GridFSFile {
 
 /** @internal */
 export interface GridFSBucketReadStreamPrivate {
+  /**
```

Comment:
I debugged the one skipped test for a bit before deciding to skip it - I added these comments as I was debugging and decided to leave them.